client/src/shared/ui/form/ToggleSwitch.js

```diff
@@ -30,7 +31,7 @@ export default function ToggleSwitch(props) {
             htmlFor={ id }>
             { label }
           </label>
-          <div className="field-wrapper">
+          <div className="field-wrapper" onChange={ onChange }>
```

Comment:
Shouldn't that be possible already via [`restProps`](https://github.com/camunda/camunda-modeler/blob/7e3b06f368aa10ced60ef8a46aca039a47c8e7db/client/src/shared/ui/form/ToggleSwitch.js#L39)?

Reply:
It is possible actually and it is a better idea.
In my new commit I deconstructed the `form` within the `restProps` and used the `submitForm` implicitly passed on with Formik.